src/components/CreateList/index.js

```diff
@@ -0,0 +1,22 @@
+import { getToken } from '@the-collab-lab/shopping-list-utils';
+import { useLocalStorage } from '../../hooks/useLocalStorage';
+import { useHistory } from 'react-router-dom';
+
+export const CreateList = () => {
+  let history = useHistory();
```

Comment:
Why here you used `let` instead of `const`? I was confused if the `history` value is manipulated in some other part of the code that justifies the use of `let`